mk/lint.mk

```
@@ -0,0 +1,18 @@
+ADD_PYTHON_ENV := source .venv/bin/activate &&
+GOLANGCI_LINT_VERSION := v1.47.2
+
+.venv:
+	python -m venv .venv && $(ADD_PYTHON_ENV)  pip install -U pip && pip install pre-commit
```

```suggestion
	python3 -m venv .venv && $(ADD_PYTHON_ENV) pip3 install -U pip
```
sorry I wrongly suggested `pip install pre-commit` on .venv rule and the usage of python3 and pip3